postgres-driver/portal_region.go

```diff
@@ -0,0 +1,11 @@
+package postgresdriver
+
+import (
+	"context"
+
+	"github.com/pokt-foundation/transaction-db/types"
+)
+
+func (d *PostgresDriver) WriteRegion(ctx context.Context, region types.PortalRegion) error {
+	return d.InsertPortalRegion(ctx, region.PortalRegionName)
```

Comment:
Does this method have to be exported? If the `WriteRegion` method is supposed to be client-facing, then this should be hidden to avoid confusing the users of the package.

Reply:
Although you are right, I can't make the insert function inside writeregion private, it is auto generated. I'll try to investigate if there's a way so sqlc generates them as Private